client/src/components/PublicRoute.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import Cookies from 'js-cookie';
+
+const PublicRoute = ({ component, ...rest }) => {
+  const accessToken = Cookies.get('accessToken');
+  if (!accessToken) {
```

Comment:
This should be based on the presence of user in the user context

Reply:
I'm having an issue, since there's a brief moment where the user doesn't exist by the time the route is encountered